### DISCUSSION TITLE
HW 4 part 1

### DISCUSSION THREAD

**Discussion by ElisDiaz46** [cooperrc/computational-mechanics#178] [Q&A]
What does this error mean?  How should  I fix it? 

![image](https://user-images.githubusercontent.com/116235922/212180067-75e5e505-d9f0-4c41-bf90-0929a8c5648e.png)

**Comment by ElisDiaz46** [comment]
![image](https://user-images.githubusercontent.com/116235922/212180193-f6e08c7b-864d-4aa8-b2a3-e679f868d080.png)

**Reply by ElisDiaz46** [reply]
Rest of the code^^^

**Comment by johnf329** [comment]
your matrix for A is incorrectly defined I think. You should have five lists of coefficients (five equations), because you have five values in b. You are using three. I don't have an example to show you because my code for that problem also outputs a different error, but i came across this problem in a different section and this was the solution.

**Reply by ElisDiaz46** [reply]
Oh okay, Thank you
